XXE Injection/README.md

```diff
@@ -559,13 +549,6 @@ Alternativly, add your payload in `xl/sharedStrings.xml`:
 <sst xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" count="10" uniqueCount="10"><si><t>&rrr;</t></si><si><t>testA2</t></si><si><t>testA3</t></si><si><t>testA4</t></si><si><t>testA5</t></si><si><t>testB1</t></si><si><t>testB2</t></si><si><t>testB3</t></si><si><t>testB4</t></si><si><t>testB5</t></si></sst>
 ```
 
-Rebuild the Excel file.
```

Comment:
I think that it actually makes sense because it is a generic explanation how an .xlsx file is updated using `7z`. Having it at the top makes these basic 7z commands easy to look up. Having it somewhere in between payloads makes it harder to find or scroll through.
Also there are maybe multiple payloads which require an update and you dont want to redundantly have the 7z command in between every payload.